### ISSUE TITLE
请求预训练模型

### ISSUE THREAD

**Issue by klauspa** [sherleens/EmotionDistributionLearning#2]

**Comment by sherleens** [comment]
The pre-trained model can be downloaded from the [Model Zoo](https://github.com/BVLC/caffe/wiki/Model-Zoo). Our trained model will be provided ASAP.

**Comment by huanmofazong** [comment]
> The pre-trained model can be downloaded from the [Model Zoo](https://github.com/BVLC/caffe/wiki/Model-Zoo). Our trained model will be provided ASAP.

Hello, can you provide the model that was trained under the Flickr_LDL dataset?Or can you provide the predicted distribution of  Flickr_LDL's test set ?I want to quote your paper in our paper
